src/chains/ethereum/transaction/src/eip2930-access-list-transaction.ts

```diff
@@ -0,0 +1,229 @@
+import {
+  Data,
+  Quantity,
+  keccak,
+  BUFFER_EMPTY,
+  BUFFER_32_ZERO,
+  RPCQUANTITY_EMPTY
+} from "@ganache/utils";
+import { Address } from "@ganache/ethereum-address";
+import type Common from "@ethereumjs/common";
+import { BN, ecsign } from "ethereumjs-util";
+import { TypedRpcTransaction } from "./rpc-transaction";
+import { encodeRange, digest } from "@ganache/rlp";
+import { RuntimeTransaction } from "./runtime-transaction";
+import {
+  EIP2930AccessListDatabasePayload,
+  EIP2930AccessListDatabaseTx,
+  TypedDatabaseTransaction
+} from "./raw";
+import { AccessList, AccessListBuffer } from "@ethereumjs/tx";
+import { AccessLists } from "./access-lists";
+import { computeInstrinsicsAccessListTx } from "./signing";
+
+const CAPABILITIES: any[] = [2718, 2930];
```

Comment:
Thoughts on using the following type in all the transaction types instead (haven't tested it)?

The `supports` function signatures would need to be updated to `(capability: number): boolean`, of course.

```suggestion
const CAPABILITIES = [2718, 2930] as const;
```

Reply:
From an offline convo: we agree this type makes no sense like this :-)